src/scripts/elasticsearch-ubuntu-install.sh

```diff
@@ -332,6 +337,17 @@ install_azure_cloud_plugin()
     fi
 }
 
+install_additional_plugins()
+{
+    log "[install_additional_plugins] Installing additional plugins"
+    for PLUGIN in $(echo $INSTALL_ADDITIONAL_PLUGINS | tr ";" "\n")
```

Comment:
Thinking about the case where a user specifies an additional plugin here the same as one already installed e.g. Shield, Marvel, Azure Cloud, etc. Should we skip those when enumerating through the additional plugins?